docs/02. getting started/09. different_url_per_locale.md

```diff
@@ -0,0 +1,27 @@
+# Different url per locale
+
+In Fork CMS 5.9.0 and up it is possible to configure different urls per locale.
+
+You can do this by replacing `site.domain` in `app/config/parameters.yml` with `site.domain.[language]`
+
+Example:
+
+    parameters:
+        database.driver:        mysql
+        …
+        site.protocol:          https
+        site.domain.en:         fork-cms.com
+        site.domain.nl:         fork-cms.be
+        site.domain.fr:         fork-cms.fr
+        site.domain.de:         fork-cms.de
+        site.default_language:  en
```

Comment:
Should we rename this to `default_locale`?

Reply:
we should in fork 6, this already exists now so we can't change it within the current version